README.md

```
@@ -147,5 +147,114 @@ Go to EC2 console, access security group section and delete security groups crea
+kubectl get pods -n three-tier
+
+
+Expected:
+
+mongo-volume-claim   Bound
+mongodb-xxxxx        1/1 Running
+
+
+## No need hosts in Values 
+
+You can add backend-service in frontend and frontend services in Ingress with extra port 80
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)